hugo/layouts/partials/paging/prev-next.html

```diff
@@ -4,8 +4,8 @@
 
 <div class="prevnext">
     <nav class="prevnext__container" aria-label="{{ T "ui_pager_name_prevnext" }}">
-        {{- if $prev -}}
-            <a class="prevnext__prev" href="{{ $prev.RelPermalink }}" aria-label="{{ T "ui_pager_prev" }} - {{ $prev.LinkTitle }}">
+        {{- if and $prev (ne $prev.Params.coming_soon true) -}}
```

Comment:
You don't want to be stuck within the prev/next navigation here. Disabled pages should not be part of the prev/next . 
It should skip the prev/next pages that are disabled and go to the next page that is not disabled.

Reply:
For nowI think: don't disable next/prev. Disabled will show greyed out in docs. 
Would be nice if we can also skip disabled pages in the prev/next nav but I think that will be a lot of work because hugo doesn't have anything build in for this.

Reply:
> I think it's best to not disable the prev/next: than you just land on a todo/coming-soon page. I think that's better than being stuck 

I like this solution.